openmetadata-ui/src/main/resources/ui/src/components/common/EntityImport/entity-import.style.less

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+@bg-white: #ffffff;
+@border-color: #6b728066;
+@check-mark-color: #28a744;
+@fail-color: #cb2531;
+
+.ant-upload.file-dragger-wrapper {
+  border-color: @border-color;
+  border-width: 2px;
+  &:not(.ant-upload-disabled) {
+    &:hover {
+      border-color: @border-color;
+    }
+  }
+  background: @bg-white;
+  height: 360px;
+  margin: 24px 48px 0;
+  width: unset;
+}
+.entity-import-container {
+  .stepper-container {
+    max-width: 450px;
+    margin: auto;
+    margin-top: 24px;
+    padding: 0;
+  }
+  .browse-text {
+    border-bottom: 1px solid;
+  }
+}
+.passed-row {
+  color: @check-mark-color;
+}
+
+.failed-row {
+  color: @fail-color;
+}
+.import-preview-footer {
+  box-shadow: 1px -2px 4px rgba(0, 0, 0, 0.12);
```

Comment:
Do we need box shadow ?

Reply:
I have kept same style as glossary import functionality, will check with Rupesh for the same